apps/examples/src/ExamplePage.tsx

```diff
@@ -153,7 +153,7 @@ function ExampleSidebarListItem({
 
 	return (
 		<li className="examples__sidebar__item" data-active={isActive}>
-			<Link to={example.path} className="examples__sidebar__item__link hoverable" />
+			<Link replace to={example.path} className="examples__sidebar__item__link hoverable" />
```

Comment:
wait why does this prevent a full page reload? i would have thought it would just disabled the back button?

Reply:
these are ajax links/dynamic links, the current ones cause a full navigation, this just uses `history.replace`

Reply:
actually, now that i write that out-loud, i'm hearing my mistake, you're right, i have no idea why this is working - this does break Back behavior.
but without `replace` it doesn't do a dynamic load... weird

Reply:
reverted https://github.com/tldraw/tldraw/pull/5724